samples/PackageMapSample/PackageMapSample.sln

```diff
@@ -0,0 +1,34 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Why is this solution needed?

Reply:
I just made it to match the pattern of the other existing samples (which each have their own sln in addition to being part of the Microsoft.DotNet.UpgradeAssistant.sln. We may not even need those slns anymore, I just left them (and added this to match) because I didn't see a compelling reason to get rid of them.

Reply:
Thinking more about it, we don't really need the sample-specific solutions anymore, so I may just get rid of those.

Reply:
Not sure the value either. If you want a samples solution, a solution filter would be a better option.